src/trialsynth/base/ground.py

```diff
@@ -1,11 +1,67 @@
 import copy
-from typing import Iterator, Optional
+import logging
+from typing import Iterator, Optional, Callable, Tuple
 
 import gilda
+from gilda.grounder import Annotation
 from indra.databases import mesh_client
 
 from .models import BioEntity
-from .util import CONDITION_NS, INTERVENTION_NS, must_override
+from .util import (
+    CONDITION_NS,
+    INTERVENTION_NS,
+    must_override,
+    suppress_logging_info,
+    suppress_warnings,
+)
+
+with suppress_logging_info():
+    import scispacy
+    import spacy
+    from scispacy.abbreviation import AbbreviationDetector
+    from scispacy.linking import EntityLinker
+
+logger = logging.getLogger(__name__)
+
+
+class Annotator:
```

Comment:
We could instead subclass Annotator to create GildaAnnotator and SpacyAnnotator and more cleanly separate the code for each.